README.md

```diff
@@ -40,16 +42,21 @@ play_routes_pinned_maven_install()
 
 This installs `rules_play_routes` to your `WORKSPACE` at the specified commit. Update the commit as needed.
 
-## Stardoc Documentation
-http://lucidsoftware.github.io/rules_play_routes/
-
-Stardoc is replacing Skydoc and is currently under development. Doc is likely going to look funny for a while.
+## Updating Third Party Dependencies
+We use [rules_jvm_external](https://github.com/bazelbuild/rules_jvm_external) to import third party dependencies. You can change dependencies in `maven_install` in `workspace.bzl` for main `rules_play_routes` implementation or `test_workspace.bzl` for testing.
```

Comment:
We use [rules_jvm_external](https://github.com/bazelbuild/rules_jvm_external) to import third party dependencies. 

To make changes to the dependencies, simply update `maven_install` in the appropriate `workspace.bzl` file (`workspace.bzl for the main `ruless_play_routes` implementation or `test_workspace.bzl` for the tests), and then update the dependency json file used by `rules_jvm_external` by running the following script:

```bash
scripts/gen-deps.sh
```
Never modify the dependencies json file directly.

Reply:
wdyt?